### DISCUSSION TITLE
Myx Philippines

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#668] [Q&A]
**Channel Name:** Myx
**Channel ID:** MyxPhilippines.ph
**Country:** Philippines
**Language:** Tagalog/English
**External Link (optional):** xxx
**Notes (optional):** Music channel in the Philippines.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ph.m3u#L20-L21
